### PR TITLE
API provides client with a collection of departments

### PR DESCRIPTION
**Pull request by AugustEnhager**
PT Link: https://www.pivotaltracker.com/story/show/180222754

## User Story
```
As an API
In order to present the admin client with departments
I would like to be able to provide an endpoint for a department index
```

## Changes Proposed
- Add an department index action, controller, factory and serializer
- Attributes `:name` and `:average_score`
- Add has_many participants association
- Return department names and their associated participants

## Screenshots
Skärmavbild 2021-11-13 kl  11 16 51
Skärmavbild 2021-11-13 kl  11 16 34

### CODE REVIEW

File: app/models/department.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Department < ApplicationRecord
+
+  validates_presence_of :name, :average_score
```
Comment (oliverochman) [inline app/models/department.rb]: premature to add the average score perhaps?

**Review by oliverochman — DISMISSED** [review]

**Review by Fraseer — COMMENTED** [review]
LGTM, can you seed some databases tho

File: app/models/department.rb (hunk 1 above)
Comment (dorianbuck) [reply app/models/department.rb]: We have removes traces of this future creep 👍🏽

File: db/migrate/20211110193730_add_similarity_to_fikas.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 class AddSimilarityToFikas < ActiveRecord::Migration[6.1]
   def change
     add_column :fikas, :similarity, :float
-    add_column :participants, :department, :string
     add_column :participants, :seniority, :integer
```
Comment (AugustEnhager) [inline db/migrate/20211110193730_add_similarity_to_fikas.rb:4]: In the discussion we talked about having seniority as boolean?

Comment (dorianbuck) [reply db/migrate/20211110193730_add_similarity_to_fikas.rb:4]: We will discuss this as a team and make a chore for any changes decided

**Pull request merged**